pkgs/applications/networking/deck/default.nix

```
@@ -0,0 +1,40 @@
+{ buildGoModule, lib, installShellFiles, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "deck";
+  version = "1.22.0";
+  sha = "7447a09";
+
+  src = fetchFromGitHub {
+    owner = "Kong";
+    repo = pname;
```

Overriding `pname` will break `src`.

```suggestion
    repo = "deck";
```